src/index.js

```diff
@@ -0,0 +1,124 @@
+import './sass/index.scss';
+import TodoList from './components/todolist';
+import deleteTodoItem from './components/utils/delete';
+import moveTodoItem from './components/utils/move';
+import LocalStorage from './components/local-storage';
+import TodoItem from './components/todoitem';
+import checkHandler from './components/utils/check';
+
+const list = document.querySelector('#list');
+const localStorageList = LocalStorage.get();
+
+const appendEvents = (ele) => {
+  //* onchange event for the input
+  const inputDescription = ele.querySelector('#todo_item_description');
+
+  inputDescription.addEventListener('change', (e) => {
+    const elementIndex = TodoList.getFromId(Number(ele.id));
+    if (e.target.value === '') {
+      TodoList.removeFromId(Number(ele.id));
+      e.target.parentNode.parentNode.parentNode.remove();
+    } else if (e.target.value !== TodoList.list[elementIndex].description) {
+      TodoList.list[elementIndex].description = e.target.value;
+      LocalStorage.update();
+    }
+  });
```

Comment:
- [ ] Kindly, I think you should change the name of the parameter "ele" in the appendEvents function to explain better his functionality. 🧮